lib/components/Form/AutoComplete/index.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const AutoComplete = ({ items, onClick }) => (
-  <ul>
+  <ul className="autocomplete">
     {
       // eslint-disable-next-line
-      items.map(item => <li onClick={onClick}>{item}</li>)
+      items.map(item => <li onClick={() => onClick(item)}>{`${item.firstname[0]} ${item.lastname[0]}`}</li>)
```

Comment:
this seems odd to use `firstname` and `lastname` - this can be used for anything

Reply:
Very good point